composer.json

```
@@ -12,13 +12,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "ext-oci8": ">=2.0.0",
-        "doctrine/dbal": ">=2.3,<2.8-dev"
+        "php": "^7.1",
```

I think it should be:

```suggestion
        "php": ">= 7.1.3",
```

Context: https://github.com/symfony/symfony/pull/36876